build-images.sh

```diff
@@ -1,16 +1,17 @@
-#/bin/bash
+#!/bin/bash
 
-mkdir -p ${VERSION}/ubuntu
-mkdir -p ${VERSION}/centos
-mkdir -p ${VERSION}/stretch
-mkdir -p ${VERSION}/stretch-slim
+if [ ! -d "$DIRECTORY" ]; then
```

Comment:
`$DIRECTORY` doesn't seem to be used anywhere else?

Reply:
true, fixed